spectrum_utils/plot.py

```diff
@@ -286,3 +430,83 @@ def mirror(
     )
 
     return ax
+
+
+def full_mirror(
```

Comment:
**thought:** Maybe something like `facet` to indicate that this is a combined figure with different sub-elements?